lib/fastlane/fast_file.rb

```diff
@@ -105,7 +105,7 @@ def platform(platform_name, &block)
 
       self.current_platform = platform_name
 
-      block.call
+      yield
```

Comment:
Indeed, I took it off for now. What about the rest of the changes. Even with regards to https://github.com/fastlane/fastlane/pull/1155 these changes still look as some reasonable styling improvements and prepare the code for rubocop `0.36.0`.